### DISCUSSION TITLE
Are we going to drop Flow support when we bump to graphql-js@16.x?

### DISCUSSION THREAD

**Discussion by gyDBD** [dotansimha/graphql-code-generator#6468] [Q&A]
Hi team, I am trying to find your plans on bumping graphql-js to 16.0.0, but wasn't able to find one. 

This is the graphql-js 16.0.0 Repalse plan: https://github.com/graphql/graphql-js/issues/2860
Seems they are going to drop flow support. Do you think it will affect graphql-codegen somehow? Will you still support flow plugins? This is very important to us since we are still using flow plugins.

**Comment by dotansimha** [comment]
I think GraphQL v16 should be shipped with Flow types, so no reason to remove it yet.
Also, if Flow types are going to be removed from GraphQL, the codegen plugins might still be useful for developers who are building projects with Flow.
